### ISSUE TITLE
Sidebar Not Updating Issue - "Github Repos : Status Updating... "

### ISSUE THREAD

**Issue by raisedadead** [alexharris/octopress-flat#5]
Hi Alex,

I have configured the _config.yml with my github @username and Twitter @username, but they don't update in the Side Bar after the site is deployed. Where as they work fine with the classic/original and other themes.

Can you please check.

**Comment by alexharris** [comment]
Hi,

I just checked on my local instance of Octopress (freshly installed from github), and it appears to be working as I would expect. The tweets themselves are not loading, but they are not loading with the "Classic" theme either, which points to an issue with either Twitter, or the twitter.html aside. 

I am attaching a screenshot of the Classic theme displaying this behavior. Is this the same issue you are experiencing?

![screen shot 2014-01-20 at 2 39 50 pm](https://f.cloud.github.com/assets/2344685/1958984/e781a8f4-8223-11e3-852f-3c35eccada24.png)

If not, can you provide a little more info about the exact problem you are experiencing?

thanks,
Alex

**Comment by raisedadead** [comment]
Hi Alex,

Sorry for the delay in reply...ya I think that's the same issue you have been talking about.
![octopress](https://f.cloud.github.com/assets/1884376/2011710/be5cb076-877e-11e3-869d-46e8c1d13297.PNG)

Please check the above screen for the behavior, that I have been getting on my end.
1. GitHub repos don't show up.
2. "Status Updating..." message remains.
3. No sign of the Twitter widget or tweets

**Comment by raisedadead** [comment]
Hi Alex,

Update: The twitter widget is now visible minus the tweets. Found that

```
default_asides: [asides/recent_posts.html, asides/github.html, asides/delicious.html, asides/pinboard.html, asides/googleplus.html]
```

is missing

```
asides/twitter.html
```

So that fixes "3. No Sign of the Twitter widget" partially, but again there aren't any tweets. I'll try and debug and keep ya posted.

Any ways thanks a ton for the awesome theme...really like the look and feel.

Have a nice one.

regards,
raisedadead.

**Comment by alexharris** [comment]
Hi raisedadead, 

Can you confirm that the missing aside was from _config.yml? I want to make sure that this was an issue with your specific installation, and not the theme itself.

**Regarding twitter not updating**, it would seem that Octopress has discontinued support for the Twitter aside, in favor of app.net. In order to display a twitter widget in the sidebar, you might have to manually create a widget via twitter's own widget creator, and embed that directly. Read this post for more information (not sure if it is out of date or not): http://blog.jmac.org/blog/2013/03/30/putting-twitter-back-into-octopress/

**Regarding Github repos**: I was able to get the sidebar widget to work fine, following the default instructions. Here is how my configuration looks, starting on line 57:

```
blog_index_asides: [asides/recent_posts.html, asides/twitter.html, asides/github.html]
post_asides: [asides/recent_posts.html, asides/twitter.html]
page_asides: [asides/recent_posts.html, asides/twitter.html]

# ----------------------- #
#   3rd Party Settings    #
# ----------------------- #

# Github repositories
github_user: alexharris
github_repo_count: 2
github_show_profile_link: true
github_skip_forks: true

# Twitter
twitter_user: alexharris6
twitter_tweet_button: true
```

Basically, all I did was add my github username, set repo count to 2, and make sure the aside was included in the asides array.

**Comment by raisedadead** [comment]
Hi Alex,

Thanks for the tip. Ya the missing `aside` was from `_config.yml` , seems it was really a missed config from my end. Sorry about that.

And the twitter tip really worked like charm..! But I am still not able to get the github repos working in the sidebar.
Will check and update you ASAP.

Thanks for all the support anyway.

regards,
raisedadead

**Comment by alexharris** [comment]
You have the github aside added in config, as well as your github username and "repo_count", as shown above?

**Comment by raisedadead** [comment]
Ya i have the github aside added in the config and, as well as `repo_count` set same as you have done in the snippet above. I also thought there might be an issue with my username and tried replacing your username, still doesn't work. I also checked the javascript for the same in a view that the Octopress v 2.0 missed the API changes but nothing your themes's Javascript are at par with the latest code.

Only thing that remains to be tested is that I am hosting the blog on github pages, I'll check with the deploy on my another website's hosting space via ftp and check.

I don't know If that's issue in resolving the scripts locations? Will let ya know.

regards,
raisedadead

**Comment by Melonbwead** [comment]
This is how i fixed it for mine

First there are two js error for the version i got pulled down.

Inside `aharris.js`

The `$(document).ready` is not closed properly. missing brackets syntax error.

Inside `octopress.js`

look for where the .map() functions being call, you need to change that to have 2 parameters instead of one. I found this fix from here http://blog.fire-development.com/2013/03/06/fixing-octopress-when-using-jquery/

want me to make a pull request ?  :smile:

**Comment by alexharris** [comment]
Hi Adrian,

Yeah feel free to make a pull request if you don't mind! 

thx,

Alex

**Comment by raisedadead** [comment]
Sorry, Just cleaning up my issues queue. Thanks for your help & support.
